Editor/Scripts/Analytics/MeshSyncObserver.cs

```diff
@@ -9,18 +9,30 @@ namespace Unity.MeshSync.Editor.Analytics {
     /// </summary>
     [InitializeOnLoad]
     internal static class MeshSyncObserverStartUp {
+        private const string SESSION_STATE_FLAG = "meshSyncAnalyticsSetupFlag";
+        private const int TIMEOUT_SECONDS = 120;
 
         private static void update() {
+            var alreadySetup = SessionState.GetBool(SESSION_STATE_FLAG, defaultValue: false);
+
+            // If the setup was already done, we still need to do it again at least once after a domain reload
+            if (alreadySetup) {
+                EditorApplication.update -= update;
+            }
```

Comment:
I suppose here we assume that loading the scene at startup vs loading the scene after returning from playmode are 2 different cases. But I wonder if this is true: you can change the scene during playmode and none of the changes will persist. So the editor will have to do something like the following:
Serialize Scene.
Go into play mode.
Exit play mode.
Remove scene from memory.
Reload scene in memory.
If we are allowing the update call to run for 2 mins after startup, should we not do the same after exiting playmode?

Reply:
There was an issue with exiting playmode, this now seems to be fixed by subscribing to EditorApplication.playModeStateChanged